### ISSUE TITLE
Detect and convert ANSI texts

### ISSUE THREAD

**Issue by pbstudent** [voyanttools/trombone#16]
bullet interpreted as ¥

Sinclair, S. & G. Rockwell. (2022). Contexts. Voyant Tools. Retrieved September 4, 2022, from https://voyant-tools.org/?query=resources&corpus=e44f6aba0a6c711b581e316dbff2ad0d&view=Contexts

services • identify human and other resources 
services ¥ identify human and other

**Comment by ajmacdonald** [comment]
See: https://tika.apache.org/1.28/api/org/apache/tika/parser/txt/CharsetDetector.html

**Comment by ajmacdonald** [comment]
Note to self: add option under Processing to force a particular encoding.

**Comment by pbstudent** [comment]
Thanks Andrew.

As promised a long time ago I am providing my URL to the bibliographic list discussing Voyant Tools:

https://dt.athabascau.ca/jspui/handle/10791/419 <https://dt.athabascau.ca/jspui/handle/10791/419>

It is not a great work but sufficed under my health conditions. Fortunately Voyant Tools helped fill the gap.

Typical responses were that Voyant Tools was complicated. I disagreed. The software is not complicated to use, rather it disguises the algorithms that make the corpus visualized. Perhaps more discussion in an advanced section of the help site for each tool could provide the supporting algorithms. I found no other application on the Internet that could break down a corpus so nicely as Voyant Tools.

Thank you.

Cheers,
Steve
